x-pack/solutions/security/plugins/security_solution/server/lists_integration/endpoint/handlers/exceptions_pre_single_list_find_handler.ts

```diff
@@ -28,7 +28,9 @@ export const getExceptionsPreSingleListFindHandler = (
 
     // Validate Trusted applications
     if (TrustedAppValidator.isTrustedApp({ listId })) {
-      await new TrustedAppValidator(endpointAppContextService, request).validatePreSingleListFind();
+      await new TrustedAppValidator(endpointAppContextService, request).validatePreSingleListFind(
+        data
+      );
       return data;
     }
 
```

Comment:
additionally to the above: i'd also suggest to rename the type `ValidatorCallback` (in line 18) to e.g. `ServerExtensionCallback`, as with this change, this callback does more than a validator

Reply:
I'm just going to delete the local type and point to `ExceptionsListPreSingleListFindServerExtension['callback']`. I'll make this chagne in the other handlers as well for consistency